### PR TITLE
Update SNMP enabling steps

### PR DESCRIPTION
**Pull request by sajinieKavindya**
$subject

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md [hunk 1]
```diff
@@ -14,13 +14,15 @@ Simple Network Management Protocol (SNMP) is an Internet-standard protocol for
...
      -  **snmp4j-agent-2.0.6.jar**
   
-2.  Enable SNMP in the `ei.toml` file, stored in the `<MI_HOME>/conf/` file by
-    adding the following entry: 
-    
-     ```toml
-     [synapse_properties]
-     'synapse.snmp.enabled'=true
-     ``` 
+2.  Enable SNMP in the `deployment.toml` file, stored in the `<MI_HOME>/conf/` file by
```
Comment (Mariangela) [inline en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md:17]: @sajinieKavindya, shall we rewrite this section like this. Please check the formatting when adding my steps.

2.  Enable SNMP.

       1.  Open the  `<MI_HOME>/conf/deployment.toml` file.
       2. Add the SNMP configuration as shown below. 
             
             Add the `synapse.snmp.community.name` entry to configure the synapse SNMP community name.

**Review by Mariangela — APPROVED** [review]
@sajinieKavindya , please incorporate my comments before merging.

File: en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md (hunk 1 above)
Comment (sajinieKavindya) [reply en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md]: Done

File: en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md [hunk 2]
```diff
@@ -14,13 +14,17 @@ Simple Network Management Protocol (SNMP) is an Internet-standard protocol for
...
-     ```toml
-     [synapse_properties]
-     'synapse.snmp.enabled'=true
-     ``` 
+    1. Open the `deployment.toml` file stored in the `<MI_HOME>/conf/`directory. 
+    2. Add the `synapse.snmp.enabled=true` entry to enable the SNMP agent. 
+    3. Add the `synapse.snmp.community.name` entry to configure the synapse SNMP community name.
+
+    ```toml
+    [synapse_properties]
```
Comment (sachithKay) [inline en/docs/observe/micro-integrator/classic-observability-metrics/snmp-monitoring.md:24]: Issue with the provided configuration. It needs to have double quotes around the property
```
[synapse_properties]
"synapse.snmp.enabled"=true
```

**Pull request merged**